### PR TITLE
fix: sharding and query for windows interop

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

File: test/sharding.spec.js [hunk 1]
```diff
@@ -84,8 +84,7 @@ describe('ShardingStore', () => {
     })
   })
 
-  // TODO: fix query prefix and orders
-  describe.skip('interface-datastore', () => {
+  describe('interface-datastore', () => {
```
Comment (diasdavid) [inline test/sharding.spec.js]: RAD!

**Comment by diasdavid** [comment]
@richardschneider seems that it just needs an update on linting :)

**Comment by dryajov** [comment]
travis is complaining about deprecated `new Buffer()` form, should be changed to `Buffer.from()` to make it happy.

**Comment by richardschneider** [comment]
@diasdavid its ready for a release!

**Pull request merged**